docs/imagecustomizer/configuration.md

```diff
@@ -1373,7 +1381,11 @@ os:
 
 Contains the configuration options for the OS.
 
-### resetBootLoaderType [string]
+## bootLoader type
```

Comment:
`bootLoader type` should be above `os type`, not in the middle of it.

Reply:
Ah, I guess I understood this time, this document really confused me, hopefully this time the change makes sense to you.